core/api/transport/impl/http/http_listener_impl.hpp

```diff
@@ -33,6 +33,10 @@ namespace kagome::api {
      */
     struct Configuration {
       Endpoint endpoint{};  ///< listener endpoint
+      Configuration() {
+        endpoint.address(boost::asio::ip::address_v4::any());
+        endpoint.port(40363);
```

Comment:
Why this number? A constant for it should be defined.

Reply:
Replaced by const